styleguide/components/41-organisms/image-collections/image-gallery/_image-gallery.scss

```diff
@@ -150,4 +150,181 @@
       padding-bottom: calc(((100% / 3) - #{$gutter-width} - .33px) / 1.6);
     }
   }
+
+  // Alternative styling
+  &--secondary {
+    .show-more__content {
+      @include theme('color', 'color-primary', 'link-color');
+
+      display: inline-block;
+      margin-top: .3rem;
+      text-align: right;
+
+      .show-more__total {
+        display: none;
+      }
+    }
+
+    // single image styling
+    .image-gallery {
+      text-align: left;
+
+      figcaption {
+        display: none;
+      }
+
+      .gallery-link {
+        @include mobile {
+          max-width: 270px;
+        }
+
+        @include phablet {
+          width: calc(100% / 2 - 1vw);
+        }
+
+        @include desktop {
+          width: calc((100% / 3 - #{$gutter-width} / 1.5) - .33px);
+          max-width: 100%;
+          margin: calc(#{$gutter-width} / 2) 0 0;
+        }
+
+        width: 100%;
+        margin: 1vw 0 $gutter-width;
+      }
+
+      // show more content: base settings
+      .image-gallery__show-more {
+        padding-bottom: 2.3em;
+
+        &::before {
+          background-color: transparent;
+        }
+      }
+    }
+
+    // multiple image styling
+    ul.image-gallery {
+      margin: 0 -1vw;
+
+      // overrule single image styling
+      .gallery-link {
+        width: 100%;
+        margin: 0;
+      }
+
+      li {
+        width: calc((100% / 1) - 2vw);
```

Comment:
Why divide 100% with 1? :P

Reply:
I think that's legacy from me. I was a bit anal and liked the logic of 100% / 3, 100% / 2 ... 100% / 1.